pkg/bme680_driver/contrib/bme680_hal.c

```diff
@@ -0,0 +1,87 @@
+
+#include <stdio.h>
+#include <string.h>
+
+#include "bme680_hal.h"
+
+#ifdef MODULE_PERIPH_I2C
+#include "periph/i2c.h"
+#endif
+#ifdef MODULE_PERIPH_SPI
+#include "periph/spi.h"
+#endif
+
+#include "xtimer.h"
+
+#ifndef BME680_SPI_SPEED
```

Comment:
This and all other parameters below should be defined in `bme680_params.h` and set in as far as possible in `bme680_params_t` as usual in RIOT. Furthermore, they should be called `BME680_PARAMS_xxx` to be consistent with usual parameter naming.